### PR TITLE
Update sigs.k8s.io/cluster-api to 3603557f

### PR DESCRIPTION
**Pull request by justinsb**
Automatic update; changelog:
  * 3603557f export getApiServerYaml (#477)
  * bbb8cdbd [clusterctl] Rename external to bootstrap and internal to target (#484)
  * e1903be6 Expose the "--kubeconfig" option in the kubeadm wrapper (#482)
  * 051f338b clusterctl/cmd: improve error message for not implemented interface (#478)
  * 32c2e4dd Fix a broken link. (#476)
  * a67bff65 Update links in the pull request template. (#473)
  * cc0efd3b README: Add implementer office hours (#474)
  * 029d2bb3 fixed errors from running: make verify (#469)
  * db54bbbb Reference the official git repo for vsphere provider (#472)
  * c8f5046f Makefile - Fix VENDOR_API_DIRS for Fedora and add openapigen to generate target (#466)
  * af7ca338 Openapi gen (#464)
  * fdbcdd0c fixed missing apiserver serviceaccount and clusterrole bindings to make apiserver work. (#462)
  * 8b0d9949 Regen bazel rules when running make depend (#465)
  * 24c7b1bc Update docs to record `etcd` requirement for all unit tests to pass (#456)
  * 213e59ef Add the google group info to the README.md (#461)
  * f17f564b Allow to use existing k8s cluster instead of minikube (#442)
  * 2ec8be38 Only log a warning if there was an error (#459)
  * 597fe239 fix wording (#457)
  * 4cd646d0 Add goflag parsing to pflags (#454)

/hold

### CODE REVIEW

**Comment by justinsb** [comment]
This is generated by a script (that I'm running manually at the moment).

Feedback welcome on PR title / PR message / commit message etc.

Unless there's some urgent need to merge this PR, the behaviour (I believe) should be that we update the PR in-place as changes are made to the upstream dep (rather than opening different PRs).  So let's leave this open and then when the next change is made upstream we can see how the update works.

File: vendor/sigs.k8s.io/cluster-api/pkg/controller/machine/controller.go [hunk 1]
```diff
@@ -133,9 +133,7 @@ func (c *MachineControllerImpl) Reconcile(machine *clusterv1.Machine) error {
 	// Machine resource created. Machine does not yet exist.
 	glog.Infof("Reconciling machine object %v triggers idempotent create.", m.ObjectMeta.Name)
 	err = c.create(m)
-	if err != nil {
-		glog.Warningf("unable to create machine %v: %v", name, err)
-	}
+	glog.Warningf("Unable to create machine %v: %v", name, err)
```
Comment (justinsb) [inline vendor/sigs.k8s.io/cluster-api/pkg/controller/machine/controller.go]: I made an error in #30 - the correct PR is https://github.com/kubernetes-sigs/cluster-api/pull/459

**Comment by roberthbailey** [comment]
> Feedback welcome on PR title / PR message / commit message etc.

Maybe automatically add a /hold in the initial PR message. Then a human can remove the hold when it's time to merge?

**Comment by roberthbailey** [comment]
Adding new comments each time a change is pushed would be sweet.

**Comment by roberthbailey** [comment]
I'm going to close this. If we are still running the automation, it should get re-opened, but it also needs to be rebased / updated anyway (which makes me think that the "automation" isn't still being used).

**Pull request closed without merging**